src/content/blog/2024-12-13-statewide-lidar-products-now-available.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Statewide Lidar Products Now Available
+author: Miriam Seely
+description: >-
+  We are thrilled to announce the completion of the Lidar baseline for the state
+  of Utah, a project that has been more than ten years in the making.
+date: '2024-12-13T17:26:48.506Z'
```

Comment:
**chore**: we'll need to update this date to have it show up after seans GPS post. I tested and we don't need to change the file names.